compiler/hie-core/src/Development/IDE/Core/Rules.hs

```diff
@@ -82,8 +81,7 @@ getFilesOfInterestRule :: Rules ()
 getFilesOfInterestRule = do
     defineEarlyCutoff $ \GetFilesOfInterest _file -> assert (null $ fromNormalizedFilePath _file) $ do
         alwaysRerun
-        Env{..} <- getServiceEnv
```

Comment:
Slightly worried that putting this in a separate definition encourages that people use it directly instead of going via the rule but I don’t feel strongly about this so your choice.

Reply:
Yeah, I was going to centralise the ofInterest rule in a future patch - but I'll do it now. I want it to be fully abstract.

Reply:
Done now

Reply:
Thanks!